CONTRIBUTING.md

```diff
@@ -1,15 +1,116 @@
 # Contributing
 
-Contributions are welcome!
+balenaSense is one of the flagship projects created and maintained by balenaLabs. Community contribution is greatly encouraged and appreciated. Use the following guidelines to successfully and sustainably contribute your hard work to take balenaSense to the next level.
 
-## Semantic versioning and commit messages
 
-The balenaSense version numbering adheres to [Semantic Versioning](http://semver.org/). The following
-header/row is required in the body of a commit message, and will cause the CI build to fail if absent:
+## Overview
+
+Make sure you checkout our [architecture][ARCHITECTURE] guide, which aims to explain how all pieces fit together. It's a good starting point for understanding how we designed and built balenaSense.
```

Comment:
I don't think we have a public architecture guide for sense, right? Perhaps we can leave this but add an issue to have it done later.

Reply:
I agree. I meant to just have this as placeholder. If it's misleading/not helpful, I can issue it and remove it. Let me bring up the architecture guide in this week's Labs call to see how soon we can populate this. Thanks @tmigone !

Reply:
yeah true. for sound we don't have a public one either lol.